csmith_testing/compile_run.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# !!!README!!!
+# The script assumes that:
+#   - file ./build/runtime.o is present
+#   - `LINKER_SCRIPT` env var is set to absolute path to linker script
+#   - PATH contains path to a directory to bin directory of llvm-valida build
+#   - gcc-multilib is installed
+#
+# The script builds csmith testcases for x86 and for delendum arch.
+# The latter is built with 3 different optimization levels.
+# Then all 4 programs are run and it's checked that their output is the same.
+#
+# At the end of a csmith program a hash of the program's state is computed and printed.
+
+set -e
+
+rm -rf build/test_* || true
+rm -rf build/log_* || true
+
+src=$1
+if [[ ! -f "$src" ]]
+then
+    echo "File does not exist"
+    exit 1
+fi
+
+echo "File under test: $src"
+
+echo "Compiling fox x86"
+
+clang -Wno-everything -std=c11 -m32 -O3 -g -nostdinc -I. -Icsmith -Icsmith/X86 $src -o build/test_x86
+
+opts="-std=c11 -target delendum -I. -Icsmith -Icsmith/Delendum"
+optsw="$opts -Wno-everything"
+
+echo "Compiling for Delendum"
+
+echo "  Compiling testcases with different opt levels"
+
+clang $optsw -O3 -c $src -o build/test_O3.o
+clang $optsw -O2 -c $src -o build/test_O2.o
```

Comment:
When I run locally, this `clang` points to the system clang which caused errors. 

So it is best to distinguish it with the system clang.

Reply:
In `run-valida-tests.sh` there is a line:

```
PATH=$(realpath ./llvm-valida/build/bin/):$(realpath ./valida/target/release/):$PATH LINKER_SCRIPT=$(realpath ./llvm-valida/valida.ld) sh -c "cd ./llvm-test-suite/csmith_testing && ./suite.sh"
```

In this line the path to our custom llvm is added at the beginning of PATH.